ros_buildfarm/templates/snippet/from_base_image.Dockerfile.em

```diff
@@ -1,6 +1,15 @@
+@[if 'base_image' in locals()]@
+FROM @base_image
+@[else]@
```

Comment:
With this additional nesting please add some indentation to the line 5, 7, and 9 to keep the template more readable:

```
@[  if arch in ['i386', 'armhf', 'arm64']]@
@[  else]@
@[  end if]@
```